mutters-ink-bot/src/main/java/com/rabidgremlin/mutters/bot/ink/SessionUtils.java

```diff
@@ -65,6 +69,8 @@ public static void saveInkStoryState(Session session, StoryState storyState)
     try
     {
       session.setAttribute(SLOT_PREFIX + "0987654321STORYSTATE1234567890", storyState.toJson());
```

Comment:
Should use **storyJson** so it does not deserialize again.

Reply:
This is the story state not the story that is getting serialized again.

Reply:
My bad :(